src/utils/error.ts

```diff
@@ -729,6 +729,14 @@ export class AuthClientErrorCode {
     code: 'user-not-disabled',
     message: 'The user must be disabled in order to bulk delete it (or you must pass force=true).',
   };
+  public static INVALID_RECAPTCHA_ACTION = {
+    code: 'invalid-recaptcha-action',
+    message: 'ReCAPTCHA must be "BLOCK".'
```

Comment:
I think we should use "reCAPTCHA" here and elsewhere.

Also, the message should be "reCAPTCHA **action** must be "BLOCK"."

Reply:
Done